src/slo/x_days_sli/sli_view_querier.py

```diff
@@ -9,8 +8,10 @@ class SLIViewQuerier(object):
 
     def __init__(self, big_query):
         self.big_query = big_query
+        self.snapshotTime = None
```

Comment:
I would move this field to query method. Just in case someone will reuse this object in different threads

Reply:
done